src/index.tsx

```diff
@@ -112,10 +120,10 @@ export default class DraggableList<
     heights: null,
   };
 
-  public getItemInstance(key: string): T {
+  public getItemInstance(key: string) {
     const ref = this._itemRefs.map.get(key);
-    if (!ref) throw new Error('key not found');
-    return ref.getTemplate();
+    if (!ref?.template) throw new Error('key not found');
```

Comment:
`template` getter should be made to always return T, so we don't need to truthy check it here

Reply:
Two reasons I think we should keep this current version:

1. If the calling code _doesn't_ pass in `instanceRef`, the `template` here will be `undefined | null`. I'd assumed that `instanceRef` wouldn't necessarily be required for calling code.
2. `Ref<T>` has its `current` type set to `T | null`.